### PR TITLE
Update Modin Get Started Readme with node and mode instruction

### PR DESCRIPTION
**Pull request by bdmoore1**
# Existing Sample Changes
## Description

In May 2021, I updated the sample README files to include instructions on how to connect to a node on DevCloud. I forgot to create a PR for the Modin sample. This PR is to add instructions for how to connect to a node on DevCloud to the Modin Getting Started sample README.

Fixes Issue# 

## External Dependencies

none

## Type of change

Please delete options that are not relevant. Add a 'X' to the one that is applicable. 

- [x ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Command Line
- [ ] oneapi-cli
- [ ] Visual Studio
- [ ] Eclipse IDE
- [ x] VSCode
- [ ] When compiling the compliler flag "-Wall -Wformat-security -Werror=format-security" was used

### CODE REVIEW

**Review by praveenkk123 — DISMISSED** [review]

**Review by tomlenth — APPROVED** [review]

**Comment by praveenkk123** [comment]
hi, there is a merge conflict. can you please fix that

**Review by praveenkk123 — APPROVED** [review]
Approve after fixing merge conflicts

**Pull request merged**
